components/ProductPhotos.tsx

```diff
@@ -0,0 +1,64 @@
+import Image from 'next/image';
+import { useState } from 'react';
+import { ProductPhotosSlider } from './ProductPhotosSlider';
+
+
+type PhotosProps={
+	img:string,
+	name:string
+}
+
+export function Photos(props:PhotosProps ) {
+	//Temporarily, until I add other photos
```

Comment:
I think better option is use array from images graphql hygraph.
When you fetch the data about product you get array of images. Provide this images to props and include it here